src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Microsoft.Azure.Search.csproj

```diff
@@ -1,24 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSearch.DataPlane.props'))" />
-  <PropertyGroup>
-    <PackageId>Microsoft.Azure.Search</PackageId>
-    <Description>Makes it easy to develop a .NET application that uses Azure Search.</Description>
-    <AssemblyTitle>Microsoft Azure Search Library</AssemblyTitle>
-    <AssemblyName>Microsoft.Azure.Search</AssemblyName>
-    <Version>9.0.1</Version>
-    <PackageReleaseNotes>This is the newest major version of the Azure Search .NET SDK, based on version 2019-05-06 of the Azure Search REST API. New in this GA version is support for Cognitive Search, complex types for richer data modeling, the Autocomplete API, and additional parsing modes for Blob indexers. See this article for help on migrating to the latest version: http://aka.ms/search-sdk-upgrade. The 9.0.1 release includes a fix for the deadlock issue (https://github.com/Azure/azure-sdk-for-net/issues/6254) in the Search API.</PackageReleaseNotes>
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="..\AssemblyInfo.Common.cs" Link="Properties\AssemblyInfo.Common.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Search.Data" Version="[9.0.1, 10.0.0)" />
```

Comment:
@brjohnstmsft why did your old project have both PackageReference and ProjectReference to these libraries? It seems like that wouldn't have even worked.

Reply:
@weshaggard This goes back to a long conversation I had with @shahabhijeet last year. At that time, we were splitting what was just a single NuGet package into four packages (three component packages, and a meta-packages that includes those three). Abhijeet suggested using package references, but I found that I couldn't build in VS without project references. After some experimentation, we determined that if you have both, the project references take precedence at build time, which is what we wanted since we develop all these packages in tandem. They are build together, versioned together, and deployed together, always, much like the OData libraries (Microsoft.Spatial, Microsoft.OData.Edm, and Microsoft.OData.Core).

Is there a better way of handling this scenario?

Reply:
Given they always ship together I would just stick with ProjectReferences and remove the PackageReference, which is what we did in this PR. I just wanted to understand what you were trying to do before I approved that change.

Reply:
As long as when we build, we get the version dependencies that we expect, I'm fine with dropping the package references.